### DISCUSSION TITLE
"Expected to find project root in current working directory."

### DISCUSSION THREAD

**Discussion by sharonanderson-dev** [community/community#57905] [Programming Help] [answered]
### Body

I tried to install a command using terminal and I ended up with this error. Here's how I did
```bash
flutter pub get PACKAGE_NAME
```

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by santosharron** [comment, marked as answer]
That's not how flutter works. You need to install just like this.
```bash
flutter pub add PACKAGE_NAME
```

**Reply by sharonanderson-dev** [reply]
Yeah, thanks for the help

**Reply by queenofcorgis** [reply]
Thanks for sharing the knowledge @santosharron 🧠 . @sharonanderson-dev and thank you for marking the answer ✅

**Comment by Suleiman775** [comment]
Expected to find project root in current working directory.
لماذا تظهر الرساله هذه
